src/trace_processor/importers/common/BUILD.gn

```diff
@@ -123,6 +122,32 @@ source_set("parser_types") {
   ]
 }
 
+source_set("args_tracker_hdr") {
```

Comment:
@LalitMaganti I let Roo code fix the dependency issues -- but maybe there's a better "utils" place or similar to move the CreateSyntheticTid helper to?...

Reply:
I don't understand why you need this at at all. Any importer code in importers/proto can depend on code in impoters/common. Why do you need to split this at all instead of just depending on common directly?

Reply:
(existing dep: common -> parser_types -> packet_sequence_state_generation_hdr)

Reply:
Please add a new header in the common folder (maybe called synthetic_tid.h) or similar and put the two static functions there.